README.md

```diff
@@ -1,5 +1,108 @@
-# Run ansible scripts
+# WSO2 Identity Server Ansible scripts
+
+This repository contains the Ansible scripts for installing and configuring WSO2 Identity Server.
+
+## Supported Operating Systems
+
+- Ubuntu 16.04 or higher
+
+## Supported Ansible Versions
+
+- Ansible 2.0.0.2
+
+## Directory Structure
+```
+.
+├── dev
+│   ├── group_vars
+│   │   └── is.yml
+│   ├── host_vars
+│   │   └── wso2is.yml
+│   └── inventory
+├── FAQ.md
+├── files
+│   ├── mysql-connector-java-5.1.45-bin.jar
+│   └── wso2is-linux-installer-x64-5.6.0.deb
+├── issue_template.md
+├── LICENSE
+├── pull_request_template.md
+├── README.md
+├── roles
+│   └── wso2is
+│       ├── tasks
+│       │   ├── customize.yml
+│       │   └── main.yml
+│       └── templates
+│           ├── carbon-home
+│           │   ├── bin
+│           │   │   └── wso2server.sh.j2
+│           │   └── repository
+│           │       ├── conf
+│           │       │   ├── axis2
+│           │       │   │   └── axis2.xml.j2
+│           │       │   ├── carbon.xml.j2
+│           │       │   ├── datasources
+│           │       │   │   └── master-datasources.xml.j2
+│           │       │   ├── identity
+│           │       │   │   └── identity.xml.j2
+│           │       │   ├── registry.xml.j2
+│           │       │   ├── tomcat
+│           │       │   │   └── catalina-server.xml.j2
+│           │       │   └── user-mgt.xml.j2
+│           │       └── deployment
+│           │           └── server
+│           │               └── eventpublishers
+│           │                   ├── IsAnalytics-Publisher-wso2event-AuthenticationData.xml.j2
+│           │                   ├── IsAnalytics-Publisher-wso2event-RoleData.xml.j2
+│           │                   ├── IsAnalytics-Publisher-wso2event-SessionData.xml.j2
+│           │                   └── IsAnalytics-Publisher-wso2event-UserData.xml.j2
+│           └── wso2is.service.j2
+└── site.yml
+```
+
+## Packs to be Copied
+
+Copy the following files to `ansible-is/files` directory.
```

Comment:
Shall we change 'ansible-is/files' to 'files' as the user is already in the cloned directory?